bundles/af-core/src/test/resources/form/formcontainer/test-page-content.json

```diff
@@ -82,6 +82,156 @@
         "visible" : false,
         "customProp": "customPropValue"
       }
+    },
+    "formcontainer-submissionView": {
```

Comment:
Where are the exporter test for these, I want to check the model json being exported, can you write that as well

Reply:
Its in FormContainerImplTest @rismehta

Reply:
Added exporters